modal/stub.py

```diff
@@ -478,6 +478,10 @@ def function(
         if _warn_parentheses_missing:
             raise InvalidError("Did you forget parentheses? Suggestion: `@stub.function()`.")
 
+        if interactive:
+            # todo(nathan): should we add a migration message here?
+            raise InvalidError("interactive is now deprecated.")
```

Comment:
yeah it would be great to use `deprecation_warning` here (and in a month or two we can use `deprecation_error` instead)

Reply:
I used the `PTYInfo` field to signal to `modal-worker` that the `INTERACTIVE_FUNCTIONS` config flag is set. I believe this is the same field that `interactive=True` uses. So my understanding is if we want to support both `INTERACTIVE_FUNCTIONS` and `interactive=True` at the same time during the deprecation warning period, we will need to change some code to make `INTERACTIVE_FUNCTIONS` and `interactive=True` distinct from each other from `modal-worker`'s perspective.

Reply:
ok – maybe go straight to `deprecation_error` then. I don't think we need a grace period in this case since it's not something used in production